### DISCUSSION TITLE
Can I send GET data via raw JSON with Postman?

### DISCUSSION THREAD

**Discussion by positivedeveloper36** [adonisjs/core#3864] [Help]
If I using Post method like this : 

![enter image description here][1]

It works. I success get data id from backend

If I using Get method like this :

![enter image description here][2]

I can't get json data. Json data is empty

My code in backend :
```
public async test({ request, response }: HttpContextContract) {
    return request.body()
}
```

I've tried in other framework and it works. I tried now on adonis and its not working

Is it possible to use the GET method in adonis for this case?

Please help. Thanks

  [1]: https://i.stack.imgur.com/VarH4.png
  [2]: https://i.stack.imgur.com/RRIQ0.png

**Comment by RomainLanz** [comment]
Hey @positivedeveloper36! 👋🏻 

Per the spec, you should not send any payload via a `GET` request.

If you want to, you can:
1. Use `POST` instead
2. Send the data using query parameters.

**Reply by positivedeveloper36** [reply]
@RomainLanz Okay. I try use adonis 4.1 it works. I success get data. But in adonis 5, it does not works. I'm just curious why in adonis 5 it doesn't work
